code/modules/mapping/mapping_helpers.dm

```diff
@@ -107,15 +107,50 @@
 	. = ..()
 	if(!mapload)
 		log_mapping("[src] spawned outside of mapload!")
-		return INITIALIZE_HINT_QDEL
+		return
```

Comment:
Why was this removed ? This is still invalid use right ?

Reply:
We still have this in code:
https://github.com/tgstation/tgstation/blob/cbdcefefea65fe27ab9e9c00f7c1f5e826d8524d/code/modules/mapping/mapping_helpers.dm#L96-L98
If we have `. = ..()` and that covers what we should be returning, should we `return .` ?

Reply:
No this is fine in that case, though arguably making it explicit is easier to read.